static/app/components/modals/autofixSetupModal.tsx

```diff
@@ -190,7 +188,7 @@ function AutofixGithubIntegrationStep({
           'Install the [link:Sentry Autofix Github App] on your Github organization or each individual repository with write permissions to enable Autofix.',
```

Comment:
```suggestion
          '[link:Install the Sentry Autofix Github App] on your Github organization or each individual repository with write permissions to enable Autofix.',
```
Given the new link is now a link to the install page,i think we should make the link encompass "install" too

Reply:
Good call. Will make this update in a followup PR since I'll be modifying that text to make it optional